packages/verification-vue-component/src/locales/de.json

```diff
@@ -98,6 +98,10 @@
                     }
                 }
             },
+            "pades": {
+                "status": "PDF signatures found",
+                "details": "This document contains PDF signatures. To verify them, download the document and open it in <a href=\"https://get.adobe.com/reader/\" target=\"_blank\">Adobe Acrobat Reader</a>."
```

Comment:
@stephalba could please translate this to German?

Reply:
PDF-Signaturen gefunden

Dieses Dokument enthält PDF-Signaturen. Um diese zu überprüfen, laden Sie das Dokument herunter und öffnen Sie es im <a href=\"https://get.adobe.com/reader/\" target=\"_blank\">Adobe Acrobat Reader</a>.

Reply:
Thanks